ntoskrnl/mm/ARM3/mdlsup.c

```diff
@@ -1681,9 +1681,26 @@ MmProbeAndLockProcessPages(IN OUT PMDL MemoryDescriptorList,
                            IN KPROCESSOR_MODE AccessMode,
                            IN LOCK_OPERATION Operation)
```

Comment:
Suggestion for the function, since you're implementing now:
```c
_IRQL_requires_max_(APC_LEVEL)
VOID
NTAPI
MmProbeAndLockProcessPages(
    _Inout_ PMDL MemoryDescriptorList,
    _In_ PEPROCESS Process,
    _In_ KPROCESSOR_MODE AccessMode,
    _In_ LOCK_OPERATION Operation)
```

Reply:
Added SAL2 annotations.